broker-util/oo-admin-chk

```diff
@@ -171,7 +171,7 @@ node_hash.each do |gear_uuid, gear_info|
     if !datastore_has_gear?(gear_uuid)
       puts "#{gear_uuid}...FAIL" if $verbose
       print_message "Gear #{gear_uuid} exists on node #{gear_info[0]} (uid: #{gear_info[1]}) but does not exist in mongo database"
-      print_message "Please see https://access.redhat.com/solutions/1171163 for more information."
+      print_message "To fix this issue, remove the gear by deleting /var/lib/openshift/#{gear_uuid} and removing the user from the node host."
```

Comment:
"removing the user from the node host"—does this refer to editing `/etc/passwd`? It might help to be more explicit here.  (You can say "or other configured name service" for the off chance that the customer is using some other NSS configuration.)